ModelService/model.py

```diff
@@ -0,0 +1,73 @@
+import json
+from kafka import KafkaConsumer, KafkaProducer
+import os
+import logging
+import sys
+
+logger = logging.getLogger("Main")
+logger.setLevel(logging.INFO)
+
+
+def process_data():
+    consumer = KafkaConsumer('rainResults', bootstrap_servers=os.getenv('KAFKA'))
+    producer = KafkaProducer(bootstrap_servers=os.getenv('KAFKA'))
+    print("Model execution service started consuming!!")
```

Comment:
I think we can add logger instead of print commands. You can mark it as an enhancement for future.